### DISCUSSION TITLE
Can't get it to auto-update...

### DISCUSSION THREAD

**Discussion by KillahWalrus** [seerge/g-helper#604] [General]
As a longtime Mac user just converting to the "dark side", I can't thank you enough for this utility. However...

I followed all instructions and have GHelper running well on my 2022 G14. However, when the little popup tells me there's a new update and asks me if I want to download it, I say yes - but then it just downloads it to the file in which I've put ghelper.exe. I then have to manually extract it and replace the old version with the new. Is this supposed to happen automatically, or am I doing it right?

Thanks!

**Comment by seerge** [comment]
@KillahWalrus if it comes from alert about autoupdate - then yes, it's supposed to download zip into same folder where .exe is located and then run a powershell command to extract it (and replace old version). I don't know why it didn't work for you :) try to wait for a new update and see if it repeats

**Reply by KillahWalrus** [reply]
Thanks for the reply! Is there anything I could do? Maybe delete and reinstall? Am I right in keeping GHelper in a folder in my Program files? Again, Mac idiot trying to learn a new system here. Thanks for your patience and thanks VERY much for this program.

**Reply by seerge** [reply]
Well, you can download older (0.81 for example) and see if it updates w/o problems. It could be that (by some reason) windows has protected folder where you have exe, and didn't allow it to be autoupated (try another folder, just like in a documents or so) :)

**Reply by KillahWalrus** [reply]
Perfect! I'll move it to documents. THanks!

**Comment by Wolper321** [comment]
It's interesting that my wife's laptop (G14) couldn't auto-update and she has to update manually all the time. The script seems to be only closing the app and that's about it. Also starting the freshly downloaded app for the first time triggers the Windows smart screen or something like that.

**Reply by ganyaaaa** [reply]
same

**Reply by Kikimahe** [reply]
Smartscreen alert is normal because Ghelper is not digitally signed with a certificate.
You must click on the button "run anyway". :)
